src/resources/professionals/components/ProjectsSection.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from 'react';
+import { ProjectResponse } from 'types';
+import { Project } from 'components/Project';
+import { Box } from '@mui/system';
+import { SectionTitle } from 'components/ui';
+import { Divider } from '@mui/material';
+
+type ProjectsSectionProps = {
+  projects: ProjectResponse[] | undefined;
+};
+
+const ProjectsSection: FC<ProjectsSectionProps> = (props) => {
+  const { projects } = props;
+
+  if (!projects) return null;
```

Comment:
We should render conditionally on the parent component.  `projects?.length > 0 && ProjectsSection`